### DISCUSSION TITLE
[API] Create a source using a custom connector through API

### DISCUSSION THREAD

**Discussion by victorthevong** [airbytehq/airbyte#32743] [Connector Ideas and Features]
### What area the feature impact?

Connectors

### Relevant Information

In airbyte docs when building a custom connector, it states (https://docs.airbyte.com/integrations/custom-connectors) after creation and publishing you can programmatically use a custom connector as a source.

It seems this is only true in airbyte OSS.

In airbyte cloud's API that is currently not available. As stated here - https://airbytehq.slack.com/archives/C021JANJ6TY/p1699913551475239

The airbyte cloud UI builder allows easy creation of api based connectors and UI functionality to create a source from those connectors already. We just want the cloud API to be exposed to do the same.

**Comment by Slyracoon23** [comment]
I came into this issue as well. The docs are not clear that airbyte OSS only supports for custom connectors.  Not Airbyte Cloud

I would like an update on when cloud will support this in the future.

**Comment by marcosmarxm** [comment]
Hello 👋 this is currently been worked on. You can check the [public roadmap](https://github.com/orgs/airbytehq/projects/37?pane=issue&itemId=45471174) the estimate time is maximum Jan-2024. Sorry the confusion in the documentation, I submitted a quick improvement to make it more clear.

[a comment by an automated account is not shown]
